README.md

```diff
@@ -16,22 +16,19 @@
 [travis]: https://travis-ci.org/mcu-tools/mcuboot
 [license]: https://github.com/mcu-tools/mcuboot/blob/main/LICENSE
 
-This is mcuboot version 1.8.0-rc1
+This is MCUboot version 1.8.0-rc1
 
-MCUboot is a secure bootloader for 32-bit MCUs. The goal of MCUboot is to
-define a common infrastructure for the bootloader, system flash layout on
-microcontroller systems, and to provide a secure bootloader that enables
-simple software upgrades.
+MCUboot is a secure bootloader for 32-bit MCUs.
+The goal of MCUboot is to define a common infrastructure for the bootloader, system flash layout on microcontroller systems, and to provide a secure bootloader that enables simple software upgrades.
```

Comment:
I'm curious what editor this makes things easier for? I find the line breaks make this a lot easier to work with, and a lot easier to read as text.

Reply:
I'm currently using notepad++ as my main editor.
I believe that having 1 full sentence per line helps mostly in identifying long sentences that might need to be rephrased (or to be split), missing dots at the end of phrases, random blank spaces, and other small fixes that help to improve the clarity of the text. It does not affect docbuilding engines or static site generators like Sphinx or Jekyll.

Of course, this is just a proposal.
If it is not accepted I'll revert the changes to the previous formatting style.
Still, it was also inconsistent as sentences ended sometimes with a dot, sometimes with a dot + a blank space, and sometimes with a dot + 2 blank spaces.

Reply:
I think, regardless of whether this is wanted or not, a large format change like this should be a separate PR. As it is, right now, it makes it hard to review this change, since there are content changes mixed in with line break changes.

Reply:
Ok, I have reimplemented the 75char limit in the pages